.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/app.log
```

Comment:
Consider ignoring the entire `logs` directory instead of just `logs/app.log` to ensure all log files are ignored. You can do this by using `logs/`.